source/includes/databases-collections/time-series.php

```diff
@@ -0,0 +1,48 @@
+<?php
+require 'vendor/autoload.php'; 
+
+$uri = getenv('MONGODB_URI') ?: throw new RuntimeException('Set the MONGODB_URI variable to your Atlas URI that connects to the sample dataset');
+$client = new MongoDB\Client($uri);
+
+// Creates a time series collection to store precipitation data
+// start-create-ts
+$db = $client->precipitation;
+
+$options = [
+    'timeseries' => [
+        'timeField' => 'timestamp',
+        'metaField' => 'location',
+        'granularity' => 'minutes'
```

Comment:
The CS is to add trailing comas on multiline arrays.
```suggestion
        'granularity' => 'minutes',
```